pkgs/development/compilers/gcc/8/default.nix

```
@@ -119,14 +119,15 @@ let version = "8.3.0";
             "--disable-decimal-float" # No final libdecnumber (it may work only in 386)
         ]));
     stageNameAddon = if crossStageStatic then "-stage-static" else "-stage-final";
-    crossNameAddon = if targetPlatform != hostPlatform then "-${targetPlatform.config}" + stageNameAddon else "";
+    crossNameAddon = if targetPlatform != hostPlatform then "${targetPlatform.config}" + stageNameAddon + "-" else "";
```

```suggestion
    crossNameAddon = if targetPlatform != hostPlatform then "${targetPlatform.config}${stageNameAddon}-" else "";
```
for consistency with `gcc7`